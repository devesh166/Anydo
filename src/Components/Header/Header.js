import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";
import IconButton from "@material-ui/core/IconButton";
import MenuIcon from "@material-ui/icons/Menu";
import Button from "@material-ui/core/Button";

const useStyles = makeStyles(theme => ({
  root: {
    flexGrow: 1
  },
  menuButton: {
    marginRight: theme.spacing(2)
  },
  title: {
    flexGrow: 1
  }
}));

// const useStyles = makeStyles(theme => ({
//   root: {
//     display: "flex"
//   },
//   rootOne: {
//     flexGrow: 1
//   },
//   appBar: {
//     transition: theme.transitions.create(["margin", "width"], {
//       easing: theme.transitions.easing.sharp,
//       duration: theme.transitions.duration.leavingScreen
//     })
//   },
//   appBarShift: {
//     width: `calc(100% - ${drawerWidth}px)`,
//     marginLeft: drawerWidth,
//     transition: theme.transitions.create(["margin", "width"], {
//       easing: theme.transitions.easing.easeOut,
//       duration: theme.transitions.duration.enteringScreen
//     })
//   },
//   menuButton: {
//     marginRight: theme.spacing(2)
//   },
//   hide: {
//     display: "none"
//   },
//   drawer: {
//     width: drawerWidth,
//     flexShrink: 0
//   },
//   drawerPaper: {
//     width: drawerWidth
//   },
//   drawerHeader: {
//     display: "flex",
//     alignItems: "center",
//     padding: theme.spacing(0, 1),
//     ...theme.mixins.toolbar,
//     justifyContent: "flex-end"
//   },
//   content: {
//     flexGrow: 1,
//     padding: theme.spacing(3),
//     transition: theme.transitions.create("margin", {
//       easing: theme.transitions.easing.sharp,
//       duration: theme.transitions.duration.leavingScreen
//     }),
//     marginLeft: -drawerWidth
//   },
//   contentShift: {
//     transition: theme.transitions.create("margin", {
//       easing: theme.transitions.easing.easeOut,
//       duration: theme.transitions.duration.enteringScreen
//     }),
//     marginLeft: 0
//   }
// }));

export default function Header(props) {
  const classes = useStyles();
  // const theme = useTheme();
  // const [open, setOpen] = React.useState(false);

  // const handleDrawerOpen = () => {
  //   if (!open) setOpen(true);
  //   else setOpen(false);
  // };

  return (
    <div>
      <div style={{ zIndex: 10, position: "relative" }}>
        <AppBar position="static">
          <Toolbar>
            <IconButton
              //   aria-label="menu"
              color="inherit"
              aria-label="open drawer"
              onClick={props.handleDrawer}
              edge="start"
              className={classes.menuButton}
            >
              <MenuIcon />
            </IconButton>
            <Typography variant="h6" className={classes.title}>
              {" "}
              AnyDo{" "}
            </Typography>
            <button onClick={props.openTaskModal}>
              {" "}
              <Typography variant="h5">
                <b>Add</b>
              </Typography>
            </button>
            {/* </Typography> */}
          </Toolbar>
        </AppBar>
      </div>
    </div>
  );
}

// import React from "react";
// import { makeStyles } from "@material-ui/core/styles";
// import AppBar from "@material-ui/core/AppBar";
// import Toolbar from "@material-ui/core/Toolbar";
// import Typography from "@material-ui/core/Typography";
// import Button from "@material-ui/core/Button";
// import IconButton from "@material-ui/core/IconButton";
// import MenuIcon from "@material-ui/icons/Menu";

// const useStyles = makeStyles(theme => ({
//   root: {
//     flexGrow: 1
//   },
//   menuButton: {
//     marginRight: theme.spacing(2)
//   },
//   title: {
//     flexGrow: 1
//   }
// }));

// export default function ButtonAppBar() {
//   const classes = useStyles();

//   return (
//     <div className={classes.root}>
//       <AppBar position="static">
//         <Toolbar>
//           <IconButton
//             edge="start"
//             className={classes.menuButton}
//             color="inherit"
//             aria-label="menu"
//           >
//             <MenuIcon />
//           </IconButton>
//           <Typography variant="h6" className={classes.title}>
//             News
//           </Typography>
//           <Button color="inherit">Login</Button>
//         </Toolbar>
//       </AppBar>
//     </div>
//   );
// }
