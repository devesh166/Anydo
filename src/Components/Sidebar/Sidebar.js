import React, { useEffect } from "react";
import clsx from "clsx";
import { makeStyles } from "@material-ui/core/styles";
import Drawer from "@material-ui/core/Drawer";
import CssBaseline from "@material-ui/core/CssBaseline";
import Toolbar from "@material-ui/core/Toolbar";
import List from "@material-ui/core/List";
import Typography from "@material-ui/core/Typography";
import Divider from "@material-ui/core/Divider";
import ListItem from "@material-ui/core/ListItem";
import ListItemIcon from "@material-ui/core/ListItemIcon";
import ListItemText from "@material-ui/core/ListItemText";
import InboxIcon from "@material-ui/icons/MoveToInbox";
import MailIcon from "@material-ui/icons/Mail";
import Card from "../Card/Card";

const drawerWidth = 240;
const useStyles = makeStyles(theme => ({
  root: {
    display: "flex"
  },
  rootOne: {
    flexGrow: 1
  },
  appBar: {
    transition: theme.transitions.create(["margin", "width"], {
      easing: theme.transitions.easing.sharp,
      duration: theme.transitions.duration.leavingScreen
    })
  },
  appBarShift: {
    width: `calc(100% - ${drawerWidth}px)`,
    marginLeft: drawerWidth,
    transition: theme.transitions.create(["margin", "width"], {
      easing: theme.transitions.easing.easeOut,
      duration: theme.transitions.duration.enteringScreen
    })
  },
  menuButton: {
    marginRight: theme.spacing(2)
  },
  hide: {
    display: "none"
  },
  drawer: {
    width: drawerWidth,
    flexShrink: 0
  },
  drawerPaper: {
    width: drawerWidth
  },
  drawerHeader: {
    display: "flex",
    alignItems: "center",
    padding: theme.spacing(0, 1),
    ...theme.mixins.toolbar,
    justifyContent: "flex-start",
    fontSize: 20
  },
  content: {
    flexGrow: 1,
    padding: theme.spacing(3),
    transition: theme.transitions.create("margin", {
      easing: theme.transitions.easing.sharp,
      duration: theme.transitions.duration.leavingScreen
    }),
    marginLeft: -drawerWidth
  },
  contentShift: {
    transition: theme.transitions.create("margin", {
      easing: theme.transitions.easing.easeOut,
      duration: theme.transitions.duration.enteringScreen
    }),
    marginLeft: 0
  }
}));
function Sidebar(props) {
  const classes = useStyles();
  //   const theme = useTheme();
  const [open, setOpen] = React.useState(false);

  //   const handleDrawerOpen = () => {

  //   };
  useEffect(() => {
    setOpen(props.toggleSidebar);
    // if (!open) setOpen(true);
    // else setOpen(false);
  }, [props.toggleSidebar]);
  return (
    <div data-testid="sidebar">
      <div style={{ zIndex: 1, position: "relative" }} className={classes.root}>
        <CssBaseline />
        {/* 
        <div
          position="fixed"
          className={clsx(classes.appBar, {
            [classes.appBarShift]: open
          })}
        >
          <Toolbar>
            <IconButton>
              <MenuIcon />
            </IconButton>
            <Typography variant="h6" noWrap>
              Persistent drawer
            </Typography>
          </Toolbar>
        </div> */}

        <Drawer
          data-testid="drawer1"
          className={classes.drawer}
          variant="persistent"
          anchor="left"
          open={open}
          classes={{
            paper: classes.drawerPaper
          }}
        >
          <div className={classes.drawerHeader}></div>
          <Divider />
          <List data-testid="drawer">
            {props.List.map((text, index) => (
              <ListItem button key={text}>
                {/* <ListItemIcon>
                    {index % 2 === 0 ? <InboxIcon /> : <MailIcon />}
                  </ListItemIcon> */}
                <ListItemText
                  data-testid={`ListBtn${index}`}
                  primary={
                    <Typography variant="h5">
                      <b>{text}</b>
                    </Typography>
                  }
                  onClick={() => {
                    props.selectedList(index);
                  }}
                />
              </ListItem>
            ))}
          </List>
          <Divider />
          <button onClick={props.addList}>
            {" "}
            <Typography variant="h5">
              <b>+ New List</b>
            </Typography>
          </button>
          {/* <List>
            {["+ New List"].map((text, index) => (
              <ListItem button key={text}>
                <ListItemIcon>
                  {index % 2 === 0 ? <InboxIcon /> : <MailIcon />}
                </ListItemIcon>
                <ListItemText primary={text} />
              </ListItem>
            ))}
          </List> */}
        </Drawer>
        <main
          className={clsx(classes.content, {
            [classes.contentShift]: open
          })}
        >
          <div className={classes.drawerHeader}>
            <h2>{props.CardTittle}</h2>
          </div>
          {props.children}
        </main>
      </div>
    </div>
  );
}

export default Sidebar;
