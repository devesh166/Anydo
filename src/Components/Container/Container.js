import React, { useState } from "react";
import Header from "../Header/Header";
import Sidebar from "../Sidebar/Sidebar";
import Card from "../Card/Card";
import cloneDeep from "clone-deep";
import AddList from "../Modal/AddList";
import AddTask from "../Modal/AddTask";
function Container() {
  const [open, setOpen] = React.useState(true);
  const handleDrawer = () => {
    if (!open) setOpen(true);
    else setOpen(false);
  };
  const [ListModal, setListModal] = useState(false);
  const [TaskModal, setTaskModal] = useState(false);
  const TempArr = [[], [], []]; //today , tomm, someday
  const [List, setList] = React.useState([
    "ALL Tasks",
    "Personal",
    "Work",
    "Grocery List"
  ]);
  const [CardTittle, setCardTittle] = React.useState(List[0]);
  const [Task, setTask] = React.useState([
    cloneDeep(TempArr),
    cloneDeep(TempArr),
    cloneDeep(TempArr),
    cloneDeep(TempArr)
  ]);
  const [ListIndex, setListIndex] = useState(0);
  const openListModal = () => {
    setListModal(true);
  };
  const closeListModal = () => {
    setListModal(false);
  };
  const openTaskModal = () => {
    setTaskModal(true);
  };
  const closeTaskModal = () => {
    setTaskModal(false);
  };
  const addList = ListName => {
    // setList(List.push(newList));

    setList([...List, ListName]);
    setTask([...Task, cloneDeep(TempArr)]);
    // addTask(0, 1, "mess");
  };

  const selectedList = index => {
    setCardTittle(List[index]);
    setListIndex(index);
    console.log(List);
  };

  const addTask = (listIndex, timeIndex, message) => {
    let temp = cloneDeep(Task);
    temp[listIndex][timeIndex].push(message);
    temp[0][timeIndex].push(message);
    console.log(temp);
    setTask(temp);
  };
  React.useEffect(() => {
    console.log(Task);
  }, [Task]);
  // console.log(Task[0][0][0]);
  return (
    <div>
      <Header handleDrawer={handleDrawer} openTaskModal={openTaskModal} />
      <AddList />
      <Sidebar
        ListTittle={"All Task"}
        selectedList={selectedList}
        toggleSidebar={open}
        List={List}
        CardTittle={CardTittle}
        addList={openListModal}
        addTask={addTask}
        Task={Task}
      >
        <AddList
          closeListModal={closeListModal}
          setModal={ListModal}
          addList={addList}
        />
        <AddTask
          setModal={TaskModal}
          TaskList={List}
          addTask={addTask}
          closeTaskModal={closeTaskModal}
        />

        <Card Task={Task[ListIndex]} />
      </Sidebar>
    </div>
  );
}

export default Container;
