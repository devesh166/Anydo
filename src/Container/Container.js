import React, { useState } from "react";
import Header from "../Components/Header/Header";
import Sidebar from "../Components/Sidebar/Sidebar";
import Card from "../Components/Card/Card";
import cloneDeep from "clone-deep";
import AddList from "../Components/Modal/AddList";
import AddTask from "../Components/Modal/AddTask";
import Anydo from "../Components/Anydo/Anydo";
function Container() {
  const [open, setOpen] = React.useState(true);
  const handleDrawer = () => {
    if (!open) setOpen(true);
    else setOpen(false);
  };
  const [ListModal, setListModal] = useState(false);
  const [TaskModal, setTaskModal] = useState(false);
  const TempArr = [[], [], []]; //today , tomm, someday
  const [List, setList] = React.useState([
    "ALL Tasks",
    "Personal",
    "Work",
    "Grocery List"
  ]);
  const [CardTittle, setCardTittle] = React.useState(List[0]);
  const [Task, setTask] = React.useState([
    cloneDeep(TempArr),
    cloneDeep(TempArr),
    cloneDeep(TempArr),
    cloneDeep(TempArr)
  ]);
  const [ListIndex, setListIndex] = useState(0);
  const openListModal = () => {
    setListModal(true);
  };
  const closeListModal = () => {
    setListModal(false);
  };
  const openTaskModal = () => {
    setTaskModal(true);
  };
  const closeTaskModal = () => {
    setTaskModal(false);
  };
  const addList = ListName => {
    // setList(List.push(newList));

    setList([...List, ListName]);
    setTask([...Task, cloneDeep(TempArr)]);
  };

  const selectedList = index => {
    setCardTittle(List[index]);
    setListIndex(index);
    console.log(List);
  };

  const addTask = (listIndex, timeIndex, message) => {
    let temp = cloneDeep(Task);
    temp[listIndex][timeIndex].push(message);
    temp[0][timeIndex].push(message);
    // console.log(temp);
    setTask(temp);
  };
  React.useEffect(() => {
    console.log(Task);
  }, [Task]);
  // console.log(Task[0][0][0]);
  return (
    <div>
      <Anydo
        //header
        handleDrawer={handleDrawer}
        openTaskModal={openTaskModal}
        //sidebar
        ListTittle={"All Task"}
        selectedList={selectedList}
        toggleSidebar={open}
        List={List}
        CardTittle={CardTittle}
        addList={openListModal}
        addTask={addTask}
        Task={Task}
        //Addlist modal
        closeListModal={closeListModal}
        setListModal={ListModal}
        addList={addList}
        //addtask modal
        setModal={TaskModal}
        TaskList={List}
        addTask={addTask}
        closeTaskModal={closeTaskModal}
        //card
        Task={Task[ListIndex]}
      ></Anydo>
    </div>
  );
}

export default Container;
