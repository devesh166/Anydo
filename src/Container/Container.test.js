import "@testing-library/jest-dom/extend-expect";
import { render, cleanup, fireEvent, screen } from "@testing-library/react";
import renderer, { create } from "react-test-renderer";
import React from "react";
import Container from "./Container";
import Sidebar from "../Components/Sidebar/Sidebar";
import Header from "../Components/Header/Header";
import { shallow, mount, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });
describe("Anydo test suit", () => {
  it("Anydo Snapshot ", () => {
    const comp = renderer.create(<Container />);
    let tree = comp.toJSON();
    expect(tree).toMatchSnapshot();
  });
  // it("Drawer open", () => {
  //   const dom = render(<Container />);
  //   expect(dom.getByTestId("drawer")).toBeInTheDocument();
  // });
});

// import "@testing-library/jest-dom/extend-expect";
// // import { render, cleanup } from "@testing-library/react";
// import React from "react";
// // import Adapter from "enzyme-adapter-react-16";
// import { render, cleanup, fireEvent, screen } from "@testing-library/react";
// // import renderer, { create } from "react-test-renderer";
// // import { shallow, mount, configure } from "enzyme";
// import Container from "./Container";

// describe("Header test suit", () => {
//   it("Anydo component ", () => {
//     const { getByTestId } = render(
//       <Container List={["ALL Tasks", "Personal", "Work", "Grocery List"]} />
//     );
//     expect(screen.getByTestId("anyDoComp")).toBeInTheDocument();

//     // fireEvent.click(getByTestId("ListBtn1"));
//   });
// });
// // configure({ adapter: new Adapter() });

// // describe("Container test suit", () => {
// //   it("Snapshot test", () => {
// //     const comp = renderer.create(<Container />);
// //     let tree = comp.toJSON();
// //     expect(tree).toMatchSnapshot();
// //   });

// //   // it("test incr btn", () => {
// //   //   let spy = jest.fn();

// //   //   const wrapper = shallow(<Biscuits />);
// //   //   const btn = wrapper.find("#btn");
// //   //   btn.simulate("click");
// //   //   expect(spy).toBeDefined();
// //   // });
// //   it("test decr btn", () => {
// //     // const component = create(<Container />);
// //     // const instance = component.root;
// //     // const button = instance.findByType("button");
// //     // button.props.onClick();
// //     // expect(button.props.children).toBe("PROCEED TO CHECKOUT");
// //   });
// // });

// // // describe("removeButton", () => {
// // //   it("should call remove on click", () => {
// // //     const mock = jest.spyOn(Bis, "incrementBiscuits");
// // //     mock.mockImplementation(() => {});
// // //     const component = shallow(<Biscuits />);
// // //     component.find("#btn").simulate("click");
// // //     expect(mock).toHaveBeenCalled();
// // //     mock.mockRestore();
// // //   });
// // // });
