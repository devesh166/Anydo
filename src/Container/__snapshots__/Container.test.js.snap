// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Anydo test suit Anydo Snapshot  1`] = `
<div>
  <div
    data-testid="anydo_comp"
  >
    <div>
      <div
        style={
          Object {
            "position": "relative",
            "zIndex": 10,
          }
        }
      >
        <header
          className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
        >
          <div
            className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
          >
            <button
              aria-label="open drawer"
              className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-2 MuiIconButton-colorInherit MuiIconButton-edgeStart"
              data-testid="icon"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                  />
                </svg>
              </span>
            </button>
            <h6
              className="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
            >
               
              AnyDo
               
            </h6>
            <button
              data-testid="btn"
              onClick={[Function]}
            >
               
              <h5
                className="MuiTypography-root MuiTypography-h5"
              >
                <b>
                  Add
                </b>
              </h5>
            </button>
          </div>
        </header>
      </div>
    </div>
    <div />
    <div />
    <div
      data-testid="sidebar"
    >
      <div
        className="makeStyles-root-99"
        style={
          Object {
            "position": "relative",
            "zIndex": 1,
          }
        }
      >
        <div
          className="MuiDrawer-root MuiDrawer-docked makeStyles-drawer-105"
          data-testid="drawer1"
        >
          <div
            className="MuiPaper-root MuiDrawer-paper makeStyles-drawerPaper-106 MuiDrawer-paperAnchorLeft MuiDrawer-paperAnchorDockedLeft MuiPaper-elevation0"
            style={
              Object {
                "visibility": "hidden",
              }
            }
          >
            <div
              className="makeStyles-drawerHeader-107"
            />
            <hr
              className="MuiDivider-root"
            />
            <ul
              className="MuiList-root MuiList-padding"
              data-testid="drawer"
            >
              <div
                aria-disabled={false}
                className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiListItemText-root"
                  data-testid="ListBtn0"
                  onClick={[Function]}
                >
                  <h5
                    className="MuiTypography-root MuiTypography-h5"
                  >
                    <b>
                      ALL Tasks
                    </b>
                  </h5>
                </div>
              </div>
              <div
                aria-disabled={false}
                className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiListItemText-root"
                  data-testid="ListBtn1"
                  onClick={[Function]}
                >
                  <h5
                    className="MuiTypography-root MuiTypography-h5"
                  >
                    <b>
                      Personal
                    </b>
                  </h5>
                </div>
              </div>
              <div
                aria-disabled={false}
                className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiListItemText-root"
                  data-testid="ListBtn2"
                  onClick={[Function]}
                >
                  <h5
                    className="MuiTypography-root MuiTypography-h5"
                  >
                    <b>
                      Work
                    </b>
                  </h5>
                </div>
              </div>
              <div
                aria-disabled={false}
                className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiListItemText-root"
                  data-testid="ListBtn3"
                  onClick={[Function]}
                >
                  <h5
                    className="MuiTypography-root MuiTypography-h5"
                  >
                    <b>
                      Grocery List
                    </b>
                  </h5>
                </div>
              </div>
            </ul>
            <hr
              className="MuiDivider-root"
            />
            <button
              onClick={[Function]}
            >
               
              <h5
                className="MuiTypography-root MuiTypography-h5"
              >
                <b>
                  + New List
                </b>
              </h5>
            </button>
          </div>
        </div>
        <main
          className="makeStyles-content-108"
        >
          <div
            className="makeStyles-drawerHeader-107"
          >
            <h2>
              ALL Tasks
            </h2>
          </div>
          <div
            className="MuiPaper-root MuiCard-root makeStyles-card-149 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <nav
              aria-label="contacts"
              className="MuiList-root makeStyles-root-153 MuiList-padding"
            >
              <div
                aria-disabled={false}
                className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiListItemIcon-root"
                >
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                  </svg>
                </div>
                <div
                  className="MuiListItemText-root"
                >
                  <h5
                    className="MuiTypography-root MuiTypography-h5"
                  >
                    <b>
                      Today
                    </b>
                  </h5>
                </div>
              </div>
            </nav>
            <nav
              aria-label="contacts"
              className="MuiList-root makeStyles-root-153 MuiList-padding"
            >
              <div
                aria-disabled={false}
                className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiListItemIcon-root"
                >
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                  </svg>
                </div>
                <div
                  className="MuiListItemText-root"
                >
                  <h5
                    className="MuiTypography-root MuiTypography-h5"
                  >
                    <b>
                      Tommorow
                    </b>
                  </h5>
                </div>
              </div>
            </nav>
            <nav
              aria-label="contacts"
              className="MuiList-root makeStyles-root-153 MuiList-padding"
            >
              <div
                aria-disabled={false}
                className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiListItemIcon-root"
                >
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                    />
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                  </svg>
                </div>
                <div
                  className="MuiListItemText-root"
                >
                  <h5
                    className="MuiTypography-root MuiTypography-h5"
                  >
                    <b>
                      Upcoming
                    </b>
                  </h5>
                </div>
              </div>
            </nav>
          </div>
        </main>
      </div>
    </div>
  </div>
</div>
`;
